introduction.md

```diff
@@ -1 +1,5 @@
-I love Ruby and JavaScript!
\ No newline at end of file
+I like these
+* R
+* Powershell
+* C#
+* .NET
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?